selectors.yml

```diff
@@ -9,6 +9,9 @@ selectors:
   420:
     selector: 2664363617261496610
     name: "ethereum-testnet-goerli-optimism-1"
+  1029:
+    selector: 4459371029167934217
+    name: "bttc-testnet"
```

Comment:
I'm thinking about `bittorrent_chain-testnet`, this would be consistent with BSC name which does use the full name. @andrevmatos wdyt?